lib/form-layout.less

```diff
@@ -63,28 +63,37 @@
 
 .ct-form {
   // Locally redefine padding to Bootstrap values for this LESS block
-  @padding-large-vertical: 10px;
-  @padding-small-vertical: 5px;
-  @padding-small-horizontal: 10px;
+  @padding-large-vertical: 1rem;
+  @padding-small-vertical: 0.5rem;
+  @padding-small-horizontal: 1.5rem;
   // Bootstrap & PatternFly use a 1px border around widgets
   @border-width: 1px;
-  // Compute widget height based on (font-size * line height) + extra
-  @widget-height: @line-height-computed + (@padding-base-vertical * 2) + (@border-width * 2);
+  @widget-height: 2.25rem; // (36px for PF4 widgets)
 
   align-self: start; // Don't vertically fill content by default
   display: grid;
   grid-gap: @padding-small-vertical @padding-small-horizontal;
   // Repeat a label that is a minimum of 4em and its control that
   // fills the remaining space by a CSS variable (default: 2)
   grid-template-columns: repeat(var(--ct-form-columns), max-content 1fr);
-  place-items: baseline stretch;
+  justify-items: stretch;
+  align-content: baseline;
 
-  // All <label> elements describing form elements in PatternFly are
+  // All <label>gg/uu elements describing form elements in PatternFly are
```

Comment:
I have fix for this comment @ #12696 :wink:

It's commit https://github.com/cockpit-project/cockpit/pull/12696/commits/0b0eaced57efb9b22b5b58a950400607de9e1aa4